integration-cli/docker_cli_build_test.go

```diff
@@ -5406,7 +5406,7 @@ func (s *DockerSuite) TestBuildShellUpdatesConfig(c *check.C) {
 
 	buildImageSuccessfully(c, name, build.WithDockerfile(`FROM `+minimalBaseImage()+`
         SHELL ["foo", "-bar"]`))
-	expected := `["foo","-bar","#(nop) ","SHELL [foo -bar]"]`
+	expected := `["foo","-bar","#(nop) SHELL [foo -bar]"]`
```

Comment:
There was an inconsistency here. Two of the three places that added a `#(nop)` comment did it with string concat, where as this added it as another item in the slice.

I'm not sure if this will cause a cache miss, but at least it's consistent now.